numerical_methods/secant_method.c

```diff
@@ -0,0 +1,84 @@
+/**
+ * @file
+ * @brief [Wikipedia link](https://en.wikipedia.org/wiki/Secant_method). Find a
+ * continuous function's root by using a succession of roots of secant lines to
+ * approximate it, starting from the given points' secant line.
```

Comment:
Could you please split this into a brief and a description tag or make a more detailed description tag

Reply:
This is a singular sentence and also as brief as it gets. I legitimately can't split it.